dev-docs/bidders/nobid.md

```diff
@@ -9,12 +9,14 @@ media_types: banner, video
 tcfeu_supported: true
 gvl_id: 816
 usp_supported: true
+gpp_supported: true
 schain_supported: true
 coppa_supported: true
-userId: criteo, unifiedId, id5Id
+userId: criteo, unifiedId, id5Id, intentIqId
 safeframes_ok: true
-sidebarType: 1
+prebid_member: true
```

Comment:
I can't find you under https://prebid.org/member-directory/ is this process ongoing?

Reply:
Not a member yet. It was a mistake.